frontend/src/metabase/components/Modal/WindowModal.tsx

```diff
@@ -126,15 +134,22 @@ export class WindowModal extends Component<WindowModalProps> {
           {isOpen && (
             <CSSTransition
               key="modal"
-              classNames="Modal"
+              classNames={{
+                appear: ModalS.ModalAppear,
+                appearActive: ModalS.ModalAppearActive,
+                enter: ModalS.ModalEnter,
+                enterActive: ModalS.ModalEnterActive,
+                exit: ModalS.ModalExit,
+                exitActive: ModalS.ModalExitActive,
+              }}
               timeout={{
                 appear: 250,
                 enter: 250,
                 exit: 250,
               }}
             >
               <div
-                className={cx(backdropClassName, backdropClassnames)}
+                className={cx(ModalS.ModalBackdrop, backdropClassnames)}
```

Comment:
Noone passes `backdropClassName`, so this better be passed directly instead of putting it in the `defaultProps` to avoid confusion.